backend/package.json

```diff
@@ -7,12 +7,14 @@
     "test": "tap --ts",
     "prestart": "npm run build",
     "start": "node dist/index.js",
-    "build": "tsc",
+    "build": "rm -rf dist && tsc",
     "lint": "eslint . --ext .ts",
-    "prefund": "npm run build",
-    "fund": "node dist/tools/fund-accounts.js",
-    "precreate-feeds": "npm run build",
-    "create-feeds": "node dist/tools/create-feeds.js"
+    "presetup": "npm run build",
+    "setup": "node dist/tools/create-feeds.js && node dist/tools/fund-accounts.js",
```

Comment:
You need to first fund accounts and then create feeds from under those accounts, otherwise accounts will not have tokens to submit an extrinsic.